Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md

```diff
@@ -93,6 +93,14 @@ You can download the latest version of Umbraco from [Our](https://our.umbraco.co
 
 If you need help upgrading your project, we have some excellent [Upgrade instructions](https://our.umbraco.com/documentation/Getting-Started/Setup/Upgrading/general) you can follow. Be thorough when upgrading, as the latest upgrade might contain breaking changes and/or updated configuration.
 
+:::note
+#### Upgrading from a version prior to Umbraco 7.6.0
+With Umbraco 7.6.0 the property editors for pickers for content, media, members and related links has been updated to store UDI instead of node ID. If you're upgrading your site from a version prior to v7.6.0 or it has been upgraded earlier you might be using these pickers even though they have been marked as obsolete. 
```

Comment:
```suggestion
With Umbraco 7.6.0 the property editors for pickers for content, media, members, and related links have been updated to store UDI instead of node ID. If you're upgrading your site from a version prior to v7.6.0 or it has been upgraded earlier you might be using these pickers even though they have been marked as obsolete. 
```